Packs/Remedy_AR/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
   "name": "Remedy AR",
```

Comment:
just add BMC here

Reply:
i think that we will need to do another cleanup if i do it, it's in the description so i think there is no need